packages/inference/src/types.ts

```diff
@@ -90,3 +90,30 @@ export type RequestArgs = BaseArgs &
 		parameters?: Record<string, unknown>;
 		accessToken?: string;
 	};
+
+export interface ProviderConfig {
+	baseUrl: string;
+	makeBody: (params: BodyParams) => unknown;
```

Comment:
I think the output type could be slightly better (`unknown` is hard to work with)

Reply:
Would `object` be better? Or `Record<string, unknown>`?

The output is directly passed to `JSON.stringify` right after.